python/ray/train/torch/train_loop_utils.py

```diff
@@ -365,6 +368,8 @@ def __init__(self, amp: bool = False):
         self.amp_is_enabled = amp
         self.scaler = GradScaler() if amp else None
         self._seed = None
+        # default is CUDADeviceManager
+        self.device_manager = get_session().device_manager
```

Comment:
Let's fallback to CUDA device manager if `get_session` or `get_session().device_manager` returns None

Reply:
Yes, it's been updated.